lib/linguist/shebang.rb

```diff
@@ -23,10 +23,10 @@ def self.call(blob, candidates)
     #
     # Returns a String or nil
     def self.interpreter(data)
-      shebang = data.lines.first
-
       # First line must start with #!
-      return unless shebang && shebang.start_with?("#!")
+      return unless data.b.match?(/\A#!/n)
```

Comment:
Could we?

```suggestion
      return unless data.b.start_with?("#!")
```

or is there an encoding issue?